components/cli/vendor/github.com/docker/docker/client/service_create.go

```diff
@@ -6,9 +6,9 @@ import (
 
 	"github.com/docker/distribution/reference"
 	"github.com/docker/docker/api/types"
-	registrytypes "github.com/docker/docker/api/types/registry"
 	"github.com/docker/docker/api/types/swarm"
 	"github.com/opencontainers/go-digest"
+	"github.com/pkg/errors"
```

Comment:
This change was brought in with PR:
* https://github.com/moby/moby/pull/33575 Plugins on swarm